### PR TITLE
Question 6

### PR DESCRIPTION
**Pull request by Gulomova**
Write a program that calculates and prints the value according to the given formula:
Q = Square root of [(2 * C * D)/H]

Following are the fixed values of C and H:
C is 50. H is 30.

D is the variable whose values should be input to your program in a comma-separated sequence.

Example
Let us assume the following comma separated input sequence is given to the program:
100,150,180

The output of the program should be:
18,22,24

Hints:
If the output received is in decimal form, it should be rounded off to its nearest value (for example, if the output received is 26.0, it should be printed as 26)
In case of input data being supplied to the question, it should be assumed to be a console input.

### CODE REVIEW

File: homework_1/questions.py [hunk 1]
```diff
@@ -478,4 +478,4 @@
 # n = int(input())
 # num = Test()
 # lst = num.generator(n)
-# print(lst)
\ No newline at end of file
+# print(lst)
```
Comment (YevheniiaSmyrnova) [inline homework_1/questions.py]: Этого изменения не должно здесь быть

File: homework_1/question_3.py [hunk 2]
```diff
@@ -4,4 +4,4 @@
 ans = {}
 for i in range (1, n+1):
     ans[i] = i * i
-print(ans)
\ No newline at end of file
+print(ans)
```
Comment (YevheniiaSmyrnova) [inline homework_1/question_3.py]: Этого изменения не должно здесь быть

File: homework_1/question_2.py [hunk 3]
```diff
@@ -18,4 +18,3 @@
     n -= 1
 
 print(factorial)
-
```
Comment (YevheniiaSmyrnova) [inline homework_1/question_2.py]: Этого изменения не должно здесь быть

File: homework_1/question_1.py [hunk 4]
```diff
@@ -2,4 +2,6 @@
 
 for i in range(2000, 3201):
     if i%7 == 0 and i%5!=0:
-        print(i, end=",")
+        print(i, end = ",")
```
Comment (YevheniiaSmyrnova) [inline homework_1/question_1.py]: Этого изменения не должно здесь быть

File: homework_1/question_6.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+C, H = 50, 30
+
+
+def calc(D):
+    return sqrt((2*C*D)/H)
+
+
+D = input().split(',')
+D = [int(i) for i in D]
+D = [calc(i) for i in D]
```
Comment (YevheniiaSmyrnova) [inline homework_1/question_6.py:14]: что делает эта функция calc?

File: homework_1/question_2.py (hunk 3 above)
Comment (Gulomova) [reply homework_1/question_2.py]: Исправила

**Review by YevheniiaSmyrnova — APPROVED** [review]

**Pull request merged**